pkg/metricstore/metricstore.go

```diff
@@ -78,6 +80,12 @@ func NewMetricStore(envVariable environment.EnvVariables) MetricStore {
 			Name:        "k8s_api_event_counter",
 			Help:        "This counter represents the number of events received from k8s API Server",
 		}),
+		addressPoolARMFallbackCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace:   PrometheusNamespace,
+			ConstLabels: constLabels,
+			Name:        "ccp_addresspool_armfallback_counter",
```

Comment:
nit: remove CCP from the name ?

Reply:
change to "update_addresspool_armfallback_counter"